klippa/src/os2.rs

```diff
@@ -0,0 +1,125 @@
+//! impl subset() for OS/2
+use crate::SubsetFlags;
+use crate::{Plan, SubsetError, SubsetError::SubsetTableError};
+use std::cmp::Ordering;
+use write_fonts::{
+    read::{
+        collections::IntSet,
+        tables::os2::{Os2, OS2_UNICODE_RANGES},
+        FontRef, TableProvider, TopLevelTable,
+    },
+    FontBuilder,
+};
+
+pub fn subset_os2(
```

Comment:
Is there equivalent harfbuzz and/or fonttools code we could reference? If we're not doing that for our subset fns perhaps we could start? - doing it for freetype has been quite helpful.